plugins.json

```diff
@@ -2155,6 +2155,56 @@
     },
     "updated_at": "2020-11-20T18:12:44.537612606Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-apps",
```

Comment:
@mickmister , do we need an update to the statik.go file?

Reply:
@catalintomai the file is no longer part of the repo, neither is the `make generate` command. I'm thinking CI does the replacement on deployment of the marketplace now

Reply:
I believe it's meant to facilitate a faster parsing of the plugins data on a cold start of the marketplace lambda function

Reply:
`go embed` is used to include the plugin list: https://github.com/mattermost/mattermost-marketplace/pull/170